uint32 msg_id
BitMaskData[] bit_masks
float64 offset
float64 scale
